spec/latest/core/index.html

```diff
@@ -1018,28 +1018,29 @@ <h3>Templated Links</h3>
       -->
     </pre>
 
-    <p>Due to fact that hydra is on top of an RDF, which is effectively a graph,
-      it may happen that a related resource (the one on the right-hand side of the
-      relation) may not be fully described in the resource's payload. In case of
-      an <i>IriTemplate</i> a client could find out a related resource should be
-      a resource carrying a <i>template</i> predicate, but no additional
-      description is available. These rules should be considered when working with
-      <i>IriTemplate</i>s (and other hypermedia resources in general):
+    <p>Due to fact that hydra is on top of an RDF, which is a graph,
+      it may happen that a related resource (an object of the relation)
+      may not be fully described in the resource's payload. In case of
+      an <i>IriTemplate</i> expected as a related resource, client may
+      discover no additional statements describing it. These rules should
+      be considered when working with <i>IriTemplate</i>s (and other
+      hypermedia resources in general):
```

Comment:
```suggestion
      be considered when working with <i>IriTemplate</i>s (and other
     resources in general):
```

How about making this apply not only to hypermedia resource but any resource?